logger_config.py

```diff
@@ -0,0 +1,17 @@
+import logging
```

Comment:
Good work here! 🚀 I'd recommend you add the resources you used to get this done to the issue/PR. This would help others understand your approach and some of the interesting concepts used here (decorators and wrappers).

Reply:
Yeah sure @janakrajchadha :D. So should I add "Resources used" here itself in the PR description ?

Reply:
Thanks for the appreciation :). This means a lot to me and would encourage me to contribute and learn more and more.

Reply:
Sorry for the late response! I'd suggest you share it on the slack channel. That should be good :)

Reply:
Yeah, sure :D